src/components/PickerHeader.vue

```diff
@@ -26,44 +26,37 @@
 export default {
   name: 'PickerHeader',
   props: {
-    config: {
-      type: Object,
-      default() {
-        return {
-          showHeader: true,
-          isRtl: false,
-          isNextDisabled: false,
-          isPreviousDisabled: false,
-        }
-      },
+    isNextDisabled: {
+      type: Boolean,
+      required: true,
     },
-    next: {
-      type: Function,
+    isPreviousDisabled: {
+      type: Boolean,
       required: true,
     },
-    previous: {
-      type: Function,
+    isRtl: {
+      type: Boolean,
       required: true,
     },
+    isUpDisabled: {
```

Comment:
I guess this isn't needed because you only use it in the slot inside the components that include the PickerHeader

Reply:
You're right, thank you. I'll remove this from `PickerHeader` and update `PickerDay` and `PickerMonth` accordingly.